### ISSUE TITLE
MongoDB daily build error

### ISSUE THREAD

**Issue by squakez** [syndesisio/syndesis#7146]
## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Documentation issue or request
</code></pre>

## Description
The embedded mongo intermittently fails to start due to the following exception:

```
06:52:38 [INFO] Running io.syndesis.connector.mongo.MongoDBConnectorCappedCollectionConsumerTest
06:52:38 05:52:38.103 [cluster-ClusterId{value='5dce3d256efa1a6e7fd5eb8b', description='null'}-localhost:27017] DEBUG org.mongodb.driver.connection - Closing connection connectionId{localValue:62, serverValue:1}
06:52:38 05:52:38.158 [main] ERROR de.flapdoodle.embed.process.runtime.AbstractProcess - failed to call start()
06:52:38 java.io.IOException: Cannot run program "/tmp/extract-44370b4c-36de-4539-a8ef-0c5ea3eff42aextractmongod": error=26, Text file busy
06:52:38 	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
06:52:38 	at de.flapdoodle.embed.process.runtime.ProcessControl.start(ProcessControl.java:213)
06:52:38 	at de.flapdoodle.embed.process.runtime.AbstractProcess.<init>(AbstractProcess.java:99)
06:52:38 	at de.flapdoodle.embed.mongo.AbstractMongoProcess.<init>(AbstractMongoProcess.java:53)
06:52:38 	at de.flapdoodle.embed.mongo.MongodProcess.<init>(MongodProcess.java:50)
06:52:38 	at de.flapdoodle.embed.mongo.MongodExecutable.start(MongodExecutable.java:44)
06:52:38 	at de.flapdoodle.embed.mongo.MongodExecutable.start(MongodExecutable.java:34)
06:52:38 	at de.flapdoodle.embed.process.runtime.Executable.start(Executable.java:108)
06:52:38 	at io.syndesis.connector.mongo.MongoDBConnectorTestSupport.startUpMongo(MongoDBConnectorTestSupport.java:67)
06:52:38 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
06:52:38 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
06:52:38 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
06:52:38 	at java.lang.reflect.Method.invoke(Method.java:498)
06:52:38 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
06:52:38 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
06:52:38 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
06:52:38 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
06:52:38 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
06:52:38 	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
06:52:38 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
06:52:38 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
06:52:38 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
06:52:38 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
06:52:38 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
06:52:38 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
06:52:38 	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
06:52:38 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
06:52:38 Caused by: java.io.IOException: error=26, Text file busy
06:52:38 	at java.lang.UNIXProcess.forkAndExec(Native Method)
06:52:38 	at java.lang.UNIXProcess.<init>(UNIXProcess.java:247)
06:52:38 	at java.lang.ProcessImpl.start(ProcessImpl.java:134)
06:52:38 	at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
06:52:38 	... 26 common frames omitted
06:52:38 05:52:38.158 [main] INFO de.flapdoodle.embed.process.runtime.AbstractProcess - construct de.flapdoodle.embed.mongo.config.MongodConfigBuilder$ImmutableMongodConfig@15763605
06:52:38 05:52:38.158 [main] DEBUG de.flapdoodle.embed.mongo.AbstractMongoProcess - try to stop mongod
06:52:38 05:52:38.159 [main] WARN de.flapdoodle.embed.mongo.runtime.Mongod - sendShutdown localhost/127.0.0.1:27017
06:52:38 java.net.ConnectException: Connection refused (Connection refused)
06:52:38 	at java.net.PlainSocketImpl.socketConnect(Native Method)
06:52:38 	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
06:52:38 	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
06:52:38 	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
06:52:38 	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
06:52:38 	at java.net.Socket.connect(Socket.java:589)
06:52:38 	at de.flapdoodle.embed.mongo.runtime.Mongod.sendShutdown(Mongod.java:88)
06:52:38 	at de.flapdoodle.embed.mongo.AbstractMongoProcess.sendStopToMongoInstance(AbstractMongoProcess.java:133)
06:52:38 	at de.flapdoodle.embed.mongo.AbstractMongoProcess.stopInternal(AbstractMongoProcess.java:107)
06:52:38 	at de.flapdoodle.embed.process.runtime.AbstractProcess.stop(AbstractProcess.java:177)
06:52:38 	at de.flapdoodle.embed.process.runtime.AbstractProcess.<init>(AbstractProcess.java:121)
06:52:38 	at de.flapdoodle.embed.mongo.AbstractMongoProcess.<init>(AbstractMongoProcess.java:53)
06:52:38 	at de.flapdoodle.embed.mongo.MongodProcess.<init>(MongodProcess.java:50)
06:52:38 	at de.flapdoodle.embed.mongo.MongodExecutable.start(MongodExecutable.java:44)
06:52:38 	at de.flapdoodle.embed.mongo.MongodExecutable.start(MongodExecutable.java:34)
06:52:38 	at de.flapdoodle.embed.process.runtime.Executable.start(Executable.java:108)
06:52:38 	at io.syndesis.connector.mongo.MongoDBConnectorTestSupport.startUpMongo(MongoDBConnectorTestSupport.java:67)
06:52:38 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
06:52:38 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
06:52:38 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
06:52:38 	at java.lang.reflect.Method.invoke(Method.java:498)
06:52:38 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
06:52:38 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
06:52:38 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
06:52:38 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
06:52:38 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
06:52:38 	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
06:52:38 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:365)
06:52:38 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:273)
06:52:38 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:238)
06:52:38 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
06:52:38 	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
06:52:38 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
06:52:38 	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
06:52:38 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
06:52:38 05:52:38.259 [main] WARN de.flapdoodle.embed.mongo.AbstractMongoProcess - could not stop mongod with db command, try next
06:52:38 05:52:38.259 [main] WARN de.flapdoodle.embed.mongo.AbstractMongoProcess - could not stop mongod, try next
06:52:38 05:52:38.259 [main] WARN de.flapdoodle.embed.mongo.AbstractMongoProcess - could not stop mongod the second time, try one last thing
06:52:38 05:52:38.260 [main] DEBUG de.flapdoodle.embed.process.io.file.Files - could delete /tmp/embedmongo-db-06239419-e82d-4844-80a7-38ffa46c6356
06:52:38 05:52:38.260 [main] WARN de.flapdoodle.embed.process.runtime.AbstractProcess - Could not delete pid file: /tmp/extract-44370b4c-36de-4539-a8ef-0c5ea3eff42aextractmongod.pid
06:52:38 05:52:38.271 [main] DEBUG de.flapdoodle.embed.process.io.file.Files - could delete /tmp/extract-44370b4c-36de-4539-a8ef-0c5ea3eff42aextractmongod
06:52:38 05:52:38.271 [main] DEBUG org.apache.camel.test.junit4.CamelTestSupport - tearDownAfterClass test
06:52:38 [ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.191 s <<< FAILURE! - in io.syndesis.connector.mongo.MongoDBConnectorCappedCollectionConsumerTest
06:52:38 [ERROR] io.syndesis.connector.mongo.MongoDBConnectorCappedCollectionConsumerTest  Time elapsed: 0.191 s  <<< ERROR!
06:52:38 java.io.IOException: Cannot run program "/tmp/extract-44370b4c-36de-4539-a8ef-0c5ea3eff42aextractmongod": error=26, Text file busy
06:52:38 Caused by: java.io.IOException: error=26, Text file busy
```

**Comment by squakez** [comment]
Probably related to the [issue described here](https://github.com/flapdoodle-oss/de.flapdoodle.embed.mongo/issues/246).

**Comment by squakez** [comment]
Closed by #7154
